packages/web/src/components/nav/desktop/NavAudio.tsx

```diff
@@ -33,13 +34,41 @@ const messages = {
   claimRewards: 'Claim Rewards'
 }
 
-const NavAudio = () => {
+type RewardsActionBubbleProps = {
+  bubbleType: BubbleType
+  onClick(): void
```

Comment:
nit: interesting, was checking where else we use this pattern and I see one other occurrence in the client code. no biggie ofc but we've been doing `onClick: () => void`. maybe this is another client styleguide thing we should have...

Reply:
I believe they both define the same type (a function returning nothing), so it's just aesthetic preference. But I can see the benefit of consistency across the codebase. Another way I've seen it done is to create an explicit type for the callback:
```
type ClickEventHandler = () => void

type RewardsActionBubbleProps = {
  onClick: ClickEventHandler
}
```

That has the benefit of being able to define the type in one place and use it in several, with the linter/transpiler complaining at you when you change the type and forget to update all the places that use it.